README.md

```diff
@@ -40,14 +40,23 @@ https://www.codementor.io/mercurial/how-to-install-node-js-on-macos-sierra-mphz4
 nvm install 8.10.0
 ```
 
-#### 4. run the backend application
+#### 4. set backend database credentials
+
+```bash
+export EEP_DATABASE_ADMIN_NAME='eep_admin'
```

Comment:
are these the actual credentials? can you send them as an email instead and just leave placeholders here?

Reply:
yes there are actual credentials ... I can email